new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporter.java

```diff
@@ -1107,35 +1198,81 @@ private static void installMonitoringExporter() {
     monitoringExporterEndToEndDir = monitoringTemp + "/samples/kubernetes/end2end/";
     String monitoringExporterVersion = Optional.ofNullable(System.getenv("MONITORING_EXPORTER_VERSION"))
         .orElse(MONITORING_EXPORTER_VERSION);
+
+    //adding ability to build monitoring exporter if branch is not master or version number does not match latest 1.1.2
+    boolean toBuildMonitoringExporter = (!monitoringExporterBranch.equalsIgnoreCase("1.1.2")
```

Comment:
Do we need to hard-code the string 1.1.2  ?

Reply:
currently the test just use env var MONITORING_EXPORTER_VERSION to download version of exporter and run tests against it, https fix is not in released version yet, so I have added this condition to build it instead of downloading webapp based on value of release ( the latest now is 1.1.2) or value for branch, if not master it should build it.